pyinfra_borgbackup/backup.sh.j2

```diff
@@ -48,12 +47,13 @@ borg create --stats --compression lzma ${DEST1}::'backup{now:%Y-%m-%d-%H}' \
         --exclude /run               \
         --exclude /lost+found        \
         --exclude /mnt               \
+        --exclude /tmp               \
         --exclude /media             \
         --exclude /var/lib/lxcfs &
 if [ -n "${DEST2-}" ]; then
   # If there is a second backup destination, create a backup there.
   # Note: not yet supported by pyinfra deploy.
-  borg create --stats --compression lzma ${DEST2}::'backup{now:%Y-%m-%d-%H}' \
+  borg create --stats --compression lzma ${DEST2}::'backup{now:%Y-%m-%d-%H}' {{ borg_args }} \
```

Comment:
Here `/` is not removed, so with default `borg_args` being `"/"`, borg is called with `/ /` as the arguments